pkgs/development/python-modules/euclid3/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+buildPythonPackage rec {
+  pname = "euclid3";
+  version = "0.01";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-JbgnpXrb/Zo/qGJeQ6vD6Qf2HeYiND5+U4SC75tG/Qs=";
+  };
+
+  pythonImportsCheck = [ pname ];
+
+  meta = with lib; {
+    description = "2D and 3D vector, matrix, quaternion and geometry module";
+    homepage = "https://pypi.org/project/euclid3/";
```

Comment:
the real linked website is dead.

Reply:
Do you mean I should add a comment about this? (the PyPI link works fine)

Reply:
Python packages which have no homepage or source are not that great